sql/core/src/test/resources/sql-tests/inputs/pgSQL/int8.sql

```diff
@@ -0,0 +1,239 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- INT8
+-- Test int8 64-bit integers.
+-- https://github.com/postgres/postgres/blob/REL_12_BETA1/src/test/regress/sql/int8.sql
```

Comment:
Could you check `REL_12_BETA2` and update this to `REL_12_BETA2`? I guess `REL_12_BETA2` is the same.

Reply:
Yes. It's same.